index.html

```diff
@@ -163,13 +170,39 @@ <h2>
       </dd>
       <dt>
         <dfn data-dfn-for="digital credential" data-local-lt=
-        "response">Presentation response</dfn>
+        "presentation response">Presentation response</dfn>
       </dt>
       <dd>
         A format that a [=holder|holder's=] software, such as a digital wallet,
         uses, via an [=digital credential/exchange protocol=], to respond to a
         [=digital credential/request data=] by a [=verifier=].
       </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential" data-local-lt=
+        "issuance request">Issuance request</dfn>
+      </dt>
+      <dd>
+        An issuance request is an offer to issue a [=digital credential=]
+        composed of an [=digital credential/credential offer=] and a [=digital
+        credential/exchange protocol=].
```

Comment:
Maybe? Includes @marcoscaceres' [suggestion](https://github.com/WICG/digital-credentials/pull/204/files#r1976433710).
```suggestion
        An issuance request is a request that a [=digital credential=] be issued
        composed of a [=digital credential/presentation request=] and a [=digital
        credential/exchange protocol=].
```

Reply:
Done!